File: Java/GADSAAD/app/src/main/java/com/dodemy/gadsaad/model/TopLearner.java
package com.dodemy.gadsaad.model;

public class TopLearner {
    private String name;
    private String country;
    private int hours;
    private String badgeUrl;


    public TopLearner() {
    }

    public TopLearner(String name, String country, int hours, String badgeUrl) {
        this.name = name;
        this.country = country;
        this.hours = hours;
        this.badgeUrl = badgeUrl;
    }

    public int getHours() {
        return hours;
    }

    public void setHours(int hours) {
        this.hours = hours;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }


    public String getBadgeUrl() {
        return badgeUrl;
    }

    public void setBadgeUrl(String badgeUrl) {
        this.badgeUrl = badgeUrl;
    }

}

File: Kotlin/GADSPROJECTAAD/app/src/main/java/com/dodemy/gadsproject_aad/model/TopLearner.kt
package com.dodemy.gadsproject_aad.model

class TopLearner {
    var name: String? = null
    var country: String? = null
    var hours = 0
    var badgeUrl: String? = null

    constructor() {}
    constructor(name: String?, country: String?, hours: Int, badgeUrl: String?) {
        this.name = name
        this.country = country
        this.hours = hours
        this.badgeUrl = badgeUrl
    }
}